prelude.dx

```diff
@@ -8,6 +8,7 @@
 Unit = %UnitType
 Type = %TyKind
 Effects = %EffKind
+Types = %LabeledRowKind
```

Comment:
Since this is not just a list of types, but really a list of fields (i.e. types with names!), how about calling it `Fields`?